### PR TITLE
Refactors window logic/state into module

### PR DESCRIPTION
**Pull request by marcgarreau**
#### What does it do?
Refactors Window logic into the windows module. 
#### Any helpful background information?
There's duplicate Window state throughout the application, where ever a new window needs to be created. The goal is to hide as much of that away as possible within the Window module.
#### Which code should the reviewer start with?
- In `main.js`, you can see how much cleaner it is to create windows.
- The refactored logic is in `/modules/windows.js`. Now all default settings are in a method called `getDefaultOptionsForType`.

### CODE REVIEW

**Comment by frozeman** [comment]
As you are already on this, it would make sense to improve the popup window logic by pre-creating 5 windows, and only reloading the content, to be able to show them faster and create a more native feeling user experience.

**Comment by marcgarreau** [comment]
@frozeman good hypotheses - I'll start digging around.

File: interface/client/templates/popupWindows/about.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
                 License {{mist.license}}<br>
                 GitHub <a href="https://github.com/ethereum/mist" target="_blank">github.com/ethereum/mist</a>
             </p>
-            <small>Copyright 2016 Ethereum Foundation</small>
+            <small>Copyright 2017 Ethereum Foundation</small>
```
Comment (shirokoff) [inline interface/client/templates/popupWindows/about.html]: It should be on store

**Comment by marcgarreau** [comment]
closing PR while rethinking strategy; will reopen this or a new PR when the dust settles.

**Pull request closed without merging**